src/pages/index.js

```diff
@@ -13,15 +13,74 @@ import {
 } from "../utils/constants.js";
 import "./index.css";
 import UserInfo from "../components/UserInfo.js";
+import Api from "../components/Api.js";
+import PopupWithConfirmation from "../components/PopupWithConfirmation.js";
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "9a60ea94-9d34-4f3e-83e3-c406b7fc4ac5",
+    "Content-Type": "application/json",
+  },
+});
+
+const handleConfirm = (card, cardId) => {
```

Comment:
Needs correction: you should also notify user about loading process by changing the button text